spec/javascripts/admin/modules/locale-switcher.spec.js

```diff
@@ -19,10 +18,6 @@ describe('GOVUK.Modules.LocaleSwitcher', function () {
           <option value="en">English</option>
         </select>
 
-        <div class="attachment-form__isbn">
-          <input id="attachment_isbn">
-        </div>
```

Comment:
> Is this no longer needed?

Strictly speaking it isn't doing anything any more. Originally the test on the ISBN input was checking the generic 'right-to-left' class was not being added where it shouldn't be. Now that the added class is more specific (e.g. 'attachment-form__title--right-to-left') it seemed to make less sense since there was no generic class that may get wrongly added.